doc/visual-programming/source/widgets/unsupervised/tsne.md

```diff
@@ -18,11 +18,11 @@ The **t-SNE** widget plots the data with a t-distributed stochastic neighbor emb
 ![](images/tSNE-stamped.png)
 
 1. [Parameters](https://opentsne.readthedocs.io/en/latest/parameters.html) for plot optimization:
-   - measure of [perplexity](http://scikit-learn.org/stable/modules/generated/sklearn.manifold.TSNE.html). It can be thought of as the balance between preserving the global and the local structure of the data.
-   - *Preserve global structure*: using tricks for preserving global structure.
-   - *Exaggeration*: parameter which increases the attractive forces between points and allows points to move around more freely, finding their nearest neighbors more easily.
-   - *PCA components*: number of PCA components to consider for initial embedding.
-   - *Normalize data*: the values are replaced with standardized values by subtracting the average value and dividing by the standard deviation.
+   - measure of [perplexity](http://scikit-learn.org/stable/modules/generated/sklearn.manifold.TSNE.html). Roughly speaking, it can be interpreted as the number of nearest neighbors to distances will be preserved from each point. Using smaller values can reveal small, local clusters, while using large values tends to reveal the broader, global relationships between data points.
```

Comment:
> interpreted as the number of nearest neighbors to distances will be preserved from each point
I don't think this sentence makes sense.

Reply:
How about, "interpreted as the number of nearest neighbors to which distances will be preserved from each point"? Or is the entire sentence too complex?

Conceptually, each point picks some k number of nearest neighbors, and only cares aobut those. For those data points, t-SNE tries to preserve distances. For all the remaining data points, it doesn't care. Is there a shorter way of phrasing that? :)

Reply:
_Roughly speaking, it can be interpreted as the number of nearest neighbors whose distances to each point will be preserved._

How about this?

Reply:
Yeah, that sounds better.